src/css/header.css

```diff
@@ -0,0 +1,504 @@
+  .header-nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+}
+
+.header-nav-list {
+  display: none;
+}
+
+@media screen and (min-width: 1280px) {
+  .header-nav-list {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 20px;
+  }
+}
+
+.header-nav-link-one {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 1.28em;
+  color: var(--black);
+  padding: 20px 0;
+}
+
+.header-nav-link-one:after {
+  content: '';
+  position: absolute;
+  width: 37px;
+  height: 2px;
+  left: 0;
+  bottom: -4px;
+  background-color: var(--green);
+  opacity: 0;
+  transition: opacity var(--animation-fast);
+}
+
+.header-nav-link-one:hover:after {
+  opacity: 1;
+}
+
+.header-nav-link-two {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 1.28em;
+  color: var(--black);
+  padding: 20px 0;
+}
+
+.header-nav-link-two:after {
+  content: '';
+  position: absolute;
+  width: 37px;
+  height: 2px;
+  left: 80px;
+  bottom: -4px;
+  background-color: var(--green);
+  opacity: 0;
+  transition: opacity var(--animation-fast);
+}
+
+.header-nav-link-two:hover:after {
+  opacity: 1;
+}
+
+.header-nav-link-three {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 1.28em;
+  color: var(--black);
+  padding: 20px 0;
+}
+
+.header-nav-link-three:after {
+  content: '';
+  position: absolute;
+  width: 37px;
+  height: 2px;
+  left: 179px;
+  bottom: -4px;
+  background-color: var(--green);
+  opacity: 0;
+  transition: opacity var(--animation-fast);
+}
+
+.header-nav-link-three:hover:after {
+  opacity: 1;
+}
+
+.header-nav-link-four {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 1.28em;
+  color: var(--black);
+  padding: 20px 0;
+}
+
+.header-nav-link-four:after {
+  content: '';
+  position: absolute;
+  width: 37px;
+  height: 2px;
+  left: 261px;
+  bottom: -4px;
+  background-color: var(--green);
+  opacity: 0;
+  transition: opacity var(--animation-fast);
+}
+
+.header-nav-link-four:hover:after {
+  opacity: 1;
+}
+
+.header-nav-link-five {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 1.28em;
+  color: var(--black);
+  padding: 20px 0;
+}
+
+.header-nav-link-five:after {
+  content: '';
+  position: absolute;
+  width: 37px;
+  height: 2px;
+  left: 336px;
+  bottom: -4px;
+  background-color: var(--green);
+  opacity: 0;
+  transition: opacity var(--animation-fast);
+}
+
+.header-nav-link-five:hover:after {
+  opacity: 1;
+}
+
+.header-socials-list {
+  display: none;
+}
+
+@media screen and (min-width: 1280px) {
+  .header-socials-list {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 12px;
+  }
+}
+
+.header-socials-item {
+  width: 44px;
+  height: 44px;
+}
+
+.header-socials-link {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 10px;
+  border: 1px solid rgba(18, 20, 23, 0.1);
```

Comment:
use color var

Reply:
had to create a new variable color "--ultra-soft-black"